packages/@uppy/svelte/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@uppy/svelte",
+  "svelte": "src/index.js",
+  "module": "dist/index.mjs",
+  "main": "dist/index.js",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "rollup -c",
+    "prepublishOnly": "npm run build",
+    "validate": "svelte-check"
+  },
+  "devDependencies": {
+    "@rollup/plugin-node-resolve": "^9.0.0",
+    "@tsconfig/svelte": "^1.0.0",
+    "@uppy/core": "file:../../@uppy/core",
+    "rollup": "^2.0.0",
+    "rollup-plugin-svelte": "^6.0.0",
+    "svelte": "^3.0.0",
+    "svelte-check": "^1.0.0",
+    "svelte-preprocess": "^4.0.0",
+    "tslib": "^2.0.0",
+    "typescript": "^3.9.3"
```

Comment:
Maybe we should move everything to root package.json, so it’s installed with root `npm install`?